app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -151,7 +151,7 @@ KT.hosts.onKatelloHostEditLoad = function(){
             });
         });
     });
-
+    $('body').off('select2:select select2:unselecting', '#content_source_id');
```

Comment:
So on load, we detach all the callbacks that might have already been there and then attach a new one, correct?

Reply:
Yeap, something like "just remove `A` before adding `A`". It's the simplest solution, but is a workaround. Ideally, the initial problem that causes duplicate calls of `onKatelloHostEditLoad` should be fixed (but I have no idea about how to do so).

Reply:
> Ideally, the initial problem that causes duplicate calls of onKatelloHostEditLoad should be fixed (but I have no idea about how to do so).

Same, same. I'd say let's go with this for now, unless anyone else has an idea that would save the day.

Should we do the same thing for all the hooks attached on L149?

Reply:
I thought so, but I don't think it did much. The main pain was in this spot.

I'd like to get some inputs from Katello maintainers as well, since they are more familiar with the flow and what's expected.